app/controllers/application_controller.rb

```diff
@@ -10,11 +10,27 @@ class NotFoundException < RuntimeError
   end
 
   before_action :doorkeeper_authorize!
+  before_action :update_log
```

Comment:
This should probably happen before `doorkeeper_authorize!` so that every action, even if (or especially if) unauthorized, gets logged. Currently the `doorkeeper_authorize!` will halt the action chain and prevent logging 401s.